SS/events

sinjini05 opened the pull request.

sinjini05 committed:
update events.html

1. remove kloc.exe
2. space after Our Events
3. change content and image in slides
4. add more logos in collaborations

sinjini05 committed:
Update events.css

sanskritikhare142 commented on css/events.css:
```diff
@@ -197,15 +197,39 @@ iframe {
     }
 }
+.newlogo img{
+    width: 169px;
```
I think you should give dimensions in either `%` or try using `@media` query so that the images are responsive. Did you check responsiveness as of now?

sinjini05 replied:
I checked it just now and its not responsive. I'll change it.

sanskritikhare142 commented on css/events.css:
```diff
@@ -197,15 +197,39 @@ iframe {
+    position:relative;
+    right:420px;
+    top:100px;
+ 
```
Use prettier to format the files, so that there are no extra spaces and code is in same format throughout.

sanskritikhare142 replied:
Did you use prettier to format the files in the recent changes?

sinjini05 replied:
no, I forgot to change the settings, my bad. I formatted the document using prettier in the most latest ones.

sanskritikhare142 commented on events.html:
```diff
@@ -125,6 +107,12 @@ <h3 class="text-center wdwd">Want to collaborate with us?</h3>
                     <div><img src="assets/events/acm_logo_tablet 1.png"></div>
                     <div><img src="assets/events/logo-girlsxtech 2.png"></div>
+                    <div class="newlogo">
+                    <div><img src="assets/events/IIClogo.png"></div></div>
```
Using relative path everywhere is better practice, there are lesser chances of your code breaking. So try using relative paths in all src tags. It goes something like `./assets/events/imagename.png`

sanskritikhare142 replied:
I don't think you require two `div` tags here.  The `div` without any class is redundant I guess.

sinjini05 committed:
update events.html

1. used relative path
2. removed unnecessary div. Let the first two divs without class remain since it was ruining the alignment otherwise of the logos.

sinjini05 committed:
update events.css

used media query

sinjini05 committed:
Add files via upload

sinjini05 committed:
Update events.html

sinjini05 committed:
Update events.css

sinjini05 committed:
update events.html

used prettier

sinjini05 committed:
update events,css

used prettier to format it.
